features/polyExplorer/src/locales/de/stories/clusterDigitalGiantsStory.json

```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Wie die großen 6 mit Daten umgehen",
```

Comment:
Nit: capitalize, and also uniform use of name or digit

Reply:
Also gonna be changed by copy pasted texts from the wiki